zulip/integrations/bridge_with_slack/bridge_with_slack_config.py

```diff
@@ -3,12 +3,20 @@
         "email": "zulip-bot@email.com",
         "api_key": "put api key here",
         "site": "https://chat.zulip.org",
-        "stream": "test here",
-        "topic": "<- slack-bridge",
     },
     "slack": {
         "username": "slack_username",
         "token": "xoxb-your-slack-token",
-        "channel": "C5Z5N7R8A -- must be channel id",
+    },
+    # Mapping between Slack channels and Zulip stream-topic's.
+    # You can specify multiple pairs.
+    "channel_mapping": {
+        # Slack channel; must be channel ID
+        "C5Z5N7R8A>": {
```

Comment:
Is the `>` here part of the Slack channel name? I wouldn't have expected that to be present.

Reply:
Fixed. I probably copied that from a `<C5Z5N7R8A>` string and accidentally included it.